Lib/test/test_signal.py

```diff
@@ -107,6 +107,10 @@ def test_interprocess_signal(self):
         script = os.path.join(dirname, 'signalinterproctester.py')
         assert_python_ok(script)
 
+    @unittest.skipUnless(
+        hasattr(signal, "valid_signals"),
+        "requires signal.valid_signals"
+    )
```

Comment:
Usually, it's formatted as two lines:

```suggestion
    @unittest.skipUnless(hasattr(signal, "valid_signals"),
                         "requires signal.valid_signals")
```

Reply:
I prefer the black-inspired formatting style.

Reply:
I prefer consistency with other tests, but it's up to you ;-) Maybe one day Python stdlib will be formatted by Black ;-)